optimism.tokenlist.json

```diff
@@ -2764,4 +2764,4 @@
     "minor": 3,
     "patch": 0
   }
-}
\ No newline at end of file
+}
```

Comment:
Hi @themandalore , thanks for submitting this PR! Can you please remove the changes to the tokenlist.json file included in this PR?

Reply:
Don't know if you can help, I've literally edited the file directly and cut and paste the actual tokenlist like 4 times and it still won't fix it for some reason....I have no idea how to figure this one out